e2e/cron-projects.txt

```diff
@@ -0,0 +1,3 @@
+repo,metadata
```

Comment:
add "# " at start ?

Reply:
It is a CSV file. The code looks for a header.